bin/run_lint.sh

```diff
@@ -17,15 +17,14 @@ PYTHON_VERSION=$(python --version)
 
 
 if [[ "${1:-}" == "--fix" ]]; then
-    echo ">>> black fix (${PYTHON_VERSION})"
-    black $FILES
+    echo ">>> ruff fix (${PYTHON_VERSION})"
+    ruff format $FILES
+    ruff check --fix $FILES
```

Comment:
Maybe this should go in a different PR? It's a bit hard to track this change down if it's mixed in with all the devcontainer stuff.